HousingRepairsOnlineApi/Properties/launchSettings.json

```diff
@@ -28,7 +28,9 @@
         "AUTHENTICATION_IDENTIFIER": "<REPLACE WITH A UNIQUE AUTHENTICATION IDENTIFIER>",
         "JWT_SECRET" : "<REPLACE WITH A SECRET>",
         "ADDRESSES_API_URL" : "https://<REPLACE WITH ADDRESSES API URL>",
-        "SCHEDULING_API_URL" : "https://<REPLACE WITH SCHEDULING API URL>"
+        "SCHEDULING_API_URL" : "https://<REPLACE WITH SCHEDULING API URL>",
+        "GOV_NOTIFY_KEY" : "https://<REPLACE WITH GOV NOTIFY API URL>",
+        "TEMPLATE_ID": "https://<REPLACE WITH GOV NOTIFY TEMPLATE ID>"

```

Comment:
Knowing that we will have multiple templates very soon, please provide a more specific name for the environment variable.

Also this environment variable is not used anywhere in these changes - see comment in Startup.cs line 62.

Reply:
Ahh, yess